Avoid installing LICENSE and README.md in the top-level directory of Python's
site-lib directory.

Prevent installation of CLI script that doesn't work. See also:
https://github.com/networktocode/netutils/issues/898

--- pyproject.toml.orig	1970-01-01 00:00:00 UTC
+++ pyproject.toml
@@ -21,8 +21,6 @@ include = [
 ]
 
 include = [
-    "LICENSE",
-    "README.md",
     "netutils/protocols.json"
 ]
 
@@ -87,9 +85,6 @@ mkdocs-python-classy = "0.1.3"
 mkdocstrings-python = "1.13.0"
 griffe = "1.1.1"
 mkdocs-python-classy = "0.1.3"
-
-[tool.poetry.scripts]
-netutils = 'netutils.cli:main'
 
 [tool.ruff]
 line-length = 120
